src/index.tsx

```diff
@@ -444,30 +445,43 @@ export default class ReactToPrint extends React.Component<IReactToPrintProps> {
                             // the current page. Some will attempt to load the page"s parent
                             // directory. These problems can cause `react-to-print` to stop without
                             // any error being thrown. To avoid such problems we simply do not
-                            // attempt to load these links. 
-                            if (node.getAttribute("href") && !node.hasAttribute("disabled")) {
-                                const newHeadEl = domDoc.createElement(node.tagName);
-
-                                // Manually re-create the node
-                                // TODO: document why cloning the node won't work? I don't recall
-                                // the reasoning behind why we do it this way
-                                // NOTE: node.attributes has NamedNodeMap type that is not an Array
-                                // and can be iterated only via direct [i] access
-                                for (let j = 0, attrLen = node.attributes.length; j < attrLen; ++j) { // eslint-disable-line max-len
-                                    const attr = node.attributes[j];
-                                    if (attr) {
-                                        newHeadEl.setAttribute(attr.nodeName, attr.nodeValue || "");
+                            // attempt to load these links.
+                            if (node.getAttribute("href")) {
+                                // Browser's don't display `disabled` `link` nodes, so we need to filter them out
+                                // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-disabled
+                                // https://caniuse.com/mdn-html_elements_link_disabled
+                                // TODO: ideally we could just filter these out on selection, by
+                                // using a selector such as: `link[rel='stylesheet']:not([disabled=true][disabled=false])`
+                                // https://stackoverflow.com/questions/27733826/css-selectors-for-excluding-by-attribute-presence
+                                // However, that doesn't seem to work. Why?
```

Comment:
In the example above, we added many `link` tags but missing the `rel="stylesheet"` attribute, so the selector can not work well.